src/components/Card/Post/PostCardList.jsx

```diff
@@ -68,4 +69,8 @@ export default PostCardList;
 
 PostCardList.propTypes = {
   keywordId: PropTypes.string.isRequired,
+  filterList: PropTypes.shape({
+    includedKeyword: PropTypes.array.isRequired,
+    excludedKeyword: PropTypes.array.isRequired,
```

Comment:
```suggestion
    includedKeyword: PropTypes.arrayOf(PropTypes.string.isRequired),
    excludedKeyword: PropTypes.arrayOf(PropTypes.string.isRequired),
```
[prop-types 문서](https://www.npmjs.com/package/prop-types)에서 
```jsx
optionalArrayOf: PropTypes.arrayOf(PropTypes.number)
```
배열의 내부 요소의 타입까지 지정할 수 있다고 명시되어 있어서, 저희도 배열의 요소로는 문자열만 들어갈 수 있기 때문에 문자열로 타입을 지정해줘도 좋을 것 같습니다!

Reply:
저도 동감합니다. PropTypes에 대해서 최대한 구체적으로 명시하는 것이 좋을 것 같네요.

Reply:
감사합니다. 하위 요소의 타입까지 세부 설정하도록 수정했어요.